### PR TITLE
Add project metadata

### PR DESCRIPTION
**Pull request by ielis**
Add "metadata" to `pom.xml` to make the project ready for Maven Central.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,40 +7,117 @@
     <groupId>org.monarchinitiative.biodownload</groupId>
     <artifactId>biodownload</artifactId>
     <version>1.0-SNAPSHOT</version>
-    <name>biodownload</name>
+    <name>BioDownload</name>
+    <description>BioDownload is a small utility library used at the Robinson lab to download HPO, MONDO, UBERON, GO ontologies.</description>
     <url>https://github.com/TheJacksonLaboratory/biodownload</url>
+    <inceptionYear>2021</inceptionYear>
+
+    <licenses>
```
Comment (ielis) [inline pom.xml:15]: @belkassaby Can you please check if the license is OK?

File: pom.xml [hunk 2]
```diff
@@ -7,40 +7,117 @@
...
+            <name>3-clause BSD</name>
+            <url>http://www.opensource.org/licenses/bsd-license.php</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Baha El Kassaby</name>
+            <email>baha.elkassaby@jax.org</email>
```
Comment (ielis) [inline pom.xml:26]: @belkassaby is it OK to include your credentials here in this way?

**Comment by ielis** [comment]
Addresses #20

File: pom.xml (hunk 2 above)
Comment (belkassaby) [reply pom.xml:26]: sure

File: pom.xml (hunk 1 above)
Comment (belkassaby) [reply pom.xml:15]: the licence is BSD 3 right?

**Review by belkassaby — APPROVED** [review]
I have no preference with regard to the licence, but is there a reason why this is BSD 3 and other projects like phenoFX is GNU ?

**Comment by belkassaby** [comment]
> I have no preference with regard to the licence, but is there a reason why this is BSD 3 and other projects like phenoFX is GNU ?

but anyways it seems these are both compatible with each other.

**Comment by ielis** [comment]
There is no specific reason for one or another. Unless there is a better way of doing this, both code bases should be open source and easy to use. Let's discuss this the next time.

**Pull request merged**